src/Setup/DevDivVsix/MicrosoftCodeAnalysisLanguageServices/Microsoft.CodeAnalysis.LanguageServices.vsmanproj

```diff
@@ -17,7 +17,7 @@
   <Target Name="BeforeBuild">
     <MSBuild Projects="MicrosoftCodeAnalysisExpressionEvaluator\Microsoft.CodeAnalysis.ExpressionEvaluator.swixproj" Targets="Build" />
     <MSBuild Projects="MicrosoftCodeAnalysisVisualStudioInteractiveComponents\Microsoft.CodeAnalysis.VisualStudioInteractiveComponents.swixproj" Targets="Build" />
-    <MSBuild Projects="MicrosoftCodeAnalysisVisualStudioInteractiveWindow\Microsoft.CodeAnalysis.VisualStudioInteractiveWindow.swixproj" Targets="Build" />
+    <MSBuild Projects="MicrosoftVisualStudioInteractiveWindow\Microsoft.VisualStudioInteractiveWindow.swixproj" Targets="Build" />
```

Comment:
Should we move this outside the MicrosoftCodeAnalysisLanguageServices folder so that its totally separate from the 'Roslyn' authoring we have in this folder?